src/CMakeLists.txt

```diff
@@ -266,9 +266,6 @@ set( MIOpen_Source
     solver/conv/conv_hip_implicit_gemm_3d_grouped_fwd_xdlops.cpp
     solver/conv/conv_hip_implicit_gemm_3d_grouped_wrw_xdlops.cpp
     solver/conv/conv_hip_implicit_gemm_3d_grouped_bwd_xdlops.cpp
-    solver/conv/conv_hip_implicit_gemm_f16f8f16_fwd_xdlops.cpp
-    solver/conv/conv_hip_implicit_gemm_f16f8f16_bwd_xdlops.cpp
-    solver/conv/conv_hip_implicit_gemm_f16f8f16_wrw_xdlops.cpp
```

Comment:
Do we need to separate these with this PR?

Might be better to separate this out into a different PR unless it's needed for this work.

Reply:
Yes. Otherwise, we need to add the split_k stuff in the gemm_f16f8f16_wrw_xdlops.cpp solver too, which is meaningless since these solvers are totally useless now because the underlying CK instances were deleted from CK already a while ago.

Reply:
Oh that's interesting.

Alright sounds good.